components/FooterSection/FooterSection.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+
+import { FooterSectionItems } from '@/components/FooterSection/FooterSection.items';
+
+import facebookIcon from '../../public/illustrations/footer/facebookIcon.svg';
+import githubIcon from '../../public/illustrations/footer/githubIcon.svg';
+import linkedinIcon from '../../public/illustrations/footer/linkedinIcon.svg';
+import twitterIcon from '../../public/illustrations/footer/twitterIcon.svg';
+
+const FooterSection: React.FC = () => {
+  const currDate = () => {
+    return new Date().getFullYear();
+  };
+
+  return (
+    <footer className="flex flex-col divide-y-2 px-4 md:px-28">
+      <div className="basis-5/6 md:flex md:justify-between  ">
+        <div className="mt-12 md:mt-16">
+          <div className="flex flex-row items-center gap-x-2 pb-6 ">
+            <span>logo</span>
+            <span className="text-xl text-gray-900  ">Untitled UI</span>
+          </div>
+          <p className="text-gray-500 md:hidden">
+            Design amazing digital experiences that create more happy in the
+            world.
+          </p>
+        </div>
+        <div className="pt-12 md:pt-16">
+          <div className="mb-12 grid grid-cols-2 grid-rows-1 gap-8 md:grid-cols-3 md:gap-48">
+            {FooterSectionItems.map((x, index) => (
+              <div key={index}>
+                <span className="text-sm text-gray-400">{x.header}</span>
+                <ul>
+                  {x.content.map((y, index) => (
+                    <li
+                      className="mt-3 text-base font-semibold text-gray-500 first:mt-4 "
+                      key={index}
+                    >
+                      <Link href="/">{y}</Link>
```

Comment:
you can add a route parameter file to "FooterSectionItems". We will add routes later.